docs/src/pages/utils/popper/popper.md

```diff
@@ -14,8 +14,10 @@ Some important features of the `Popper` component:
 - 📦 Less than [10 KB gzipped](/size-snapshot).
 - The children is [`Portal`](/utils/portal/) to the body of the document to avoid rendering problems.
```

Comment:
Slightly unrelated but just a note, I think this needs to be reworded.

Reply:
👍 rendering problems, I would be more specific: overflow.